data/outfits.txt

```diff
@@ -137,7 +137,7 @@ outfit "Large Radar Jammer"
 	"mass" 14
 	"outfit space" -14
 	"radar jamming" 10
-	description "This high-power radar jammer will make it much more difficult for missiles that rely on radar to track you. It will not affect missiles that track you optically or that look for your ship's heat signature."
+	description "This high-powered jammer will make it much more difficult for radar-guided missiles to track you. It is even powerful enough to occasionally cause them to fly off in random directions. Like its smaller cousin, this jammer's effectiveness diminishes as the distance to the missile increases."
```

Comment:
I don't think it's necessary for the jammer descriptions to basically just repeat what the `radar jamming` tooltip says.

Reply:
It doesn't just restate the tooltip; it now mentions the haywire behavior.